src/components/EventList/EventList.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import "./EventList.scss"
+
+const eventsData = [
+  {
+    date: "January 1, 2020",
+    title: "The SE Coffee House",
+    body: "Come out for a night of music, a stage full of talent and an atmosphere of good vibes."
+  },
+  {
+    date: "January 1, 2020",
+    title: "SE eSports Tourney",
+    body: "Lockdown blues? Join us online to prove your skills, make new friends and play through the night."
+  },
+  {
+    date: "January 1, 2020",
+    title: "Club Penguin Squad Up",
+    body: "Waddle Waddle. Join us for a penguin party. Full SEnd south pole style."
+  }
+]
+
+function Event(props) {
```

Comment:
Why not an ES6 () => {} instead?

Reply:
is there more to the ES6 pattern, other than using these: `var Event(props) => { return ... }`

Reply:
meh, but I don't care about it enough to enforce any standard.